public/components/integrations/components/added_integration.tsx

```diff
@@ -27,11 +27,24 @@ import React, { useEffect, useState } from 'react';
 import _ from 'lodash';
 import { PanelTitle } from '../../trace_analytics/components/common/helper_functions';
 import { ASSET_FILTER_OPTIONS } from '../../../../common/constants/integrations';
-import { INTEGRATIONS_BASE, OBSERVABILITY_BASE } from '../../../../common/constants/shared';
+import { INTEGRATIONS_BASE } from '../../../../common/constants/shared';
 import { DeleteModal } from '../../common/helpers/delete_modal';
 import { AddedIntegrationProps } from './integration_types';
 import { useToast } from '../../../../public/components/common/toast';
 
+export const IntegrationHealthBadge = ({ status }: { status?: string }) => {
+  switch (status) {
+    case undefined:
+      return <EuiHealth color="warning">Unknown</EuiHealth>;
+    case 'available':
+      return <EuiHealth color="success">Active</EuiHealth>;
+    case 'partially-available':
+      return <EuiHealth color="warning">Partially Available</EuiHealth>;
+    default:
+      return <EuiHealth color="danger">Critical</EuiHealth>;
```

Comment:
not a blocker: Is there any documentation or comment where i can learn about how we determine the status? 
As a user I would like to learn more about what statuses represent

Reply:
We documented it in the design somewhere but I don't think we've added it to OSS docs, I'll see about finding a home for the info